### ISSUE TITLE
Tune GCP resources

### ISSUE THREAD

**Issue by javfg** [opentargets/issues#3650]
We need to tune:

* Autoscaling policies, I have a feeling they are too aggressive at the moment. ETL doesn't even have one.
* SSD sizes in most places, including PIS.
* Machine sizes.

I would delay this until we are able to label the whole set of resources a pipeline run takes, so we understand what is best to spend time tuning.

**Comment by javfg** [comment]
This requires #3646 and maybe #3647.

**Comment by d0choa** [comment]
The @javfg mentioned tickets are closed, so we might want to scope this for Q4 this year?

**Comment by javfg** [comment]
Related to https://github.com/opentargets/issues/issues/3655. This ticket would include that one, as having per-step disk size would solve it too.

I would say this goes under a discussion @project-defiant and I had where we both agreed it would be good to model a `step` object in the orchestrator. That would allow us to organize the code much better and to have these step-specific settings in a very neat way.

**Comment by d0choa** [comment]
@javfg do you think we need any specific actions now that we have the Step concept?

**Comment by javfg** [comment]
Yes, we could implement more fine-grained storage for PIS machines, etc. But I'd say it's not a priority at all. Feel free to close until we decide to focus on that.
